recipe/meta.yaml

```diff
@@ -51,6 +53,8 @@ requirements:
   host:
     - pip
     - python
+    - scikit-build
+    - scikit-build-core
```

Comment:
Is there a reason to add both `scikit-build` & `scikit-build-core`?

Thought it was one or the other. IIUC `scikit-build-core` is [replacing `scikit-build`]( https://scikit-build-core.readthedocs.io/en/latest/#scikit-build-core )